src/funcx_common/messagepack/message_types/task_cancel.py

```diff
@@ -0,0 +1,8 @@
+from .base import Message, meta
+
+
+@meta(message_type="task_cancel")
+class TaskCancel(Message):
+    # TODO: this class is a stub
+    # we need to determine fields for this message type
+    ...
```

Comment:
Definitely need to have a `task_id` in here as a field.

Reply:
Okay, I can add that right now. I'll do a fresh commit, rather than amending, for easier review.

Reply:
Just did this. Let me know if you see any issue with the change, but otherwise I'll assume it's good.